tests/backup/backup_helper.go

```diff
@@ -121,7 +126,7 @@ func getPXNamespace() string {
 	if namespace != "" {
 		return namespace
 	}
-	return storkDeploymentNamespace
+	return defaultStorkDeploymentNamespace
```

Comment:
I think you can replace this function with

`pxNamespace, err := Inst().V.GetVolumeDriverNamespace()`

Reply:
The function is not used in my testcase, will handle in different PR for that particular testcase.